components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/inbound/saml/SAMLInboundFunctions.java

```diff
@@ -171,6 +126,26 @@ public static InboundAuthenticationRequestConfig createSAMLInbound(
             serviceProvider.setCertificateContent(base64Encode(samlssoServiceProviderDTO.getCertificateContent()));
         }
 
+        return returnSAMLInbound(samlssoServiceProviderDTO);
+    }
+
+    public static InboundAuthenticationRequestConfig updateSAMLInbound(
+            ServiceProvider serviceProvider, String currentIssuer, SAML2Configuration saml2Configuration) {
+
+        SAMLSSOServiceProviderDTO samlssoServiceProviderDTO = updateSamlSSoServiceProviderDTO(saml2Configuration,
+                currentIssuer);
+
+        // Set certificate if available.
+        if (samlssoServiceProviderDTO.getCertificateContent() != null) {
+            serviceProvider.setCertificateContent(base64Encode(samlssoServiceProviderDTO.getCertificateContent()));
+        }
+
+        return returnSAMLInbound(samlssoServiceProviderDTO);
+    }
+
+    private static InboundAuthenticationRequestConfig returnSAMLInbound(
+            SAMLSSOServiceProviderDTO samlssoServiceProviderDTO) {
```

Comment:
Method name should explain the functionality.

Reply:
Changed as `createInboundAuthenticationRequestConfig`